gengokit/template/template.go

```diff
@@ -94,7 +94,7 @@ func nameServiceNameCliClientClient_mainGotemplate() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "NAME-service/NAME-cli-client/client_main.gotemplate", size: 2788, mode: os.FileMode(420), modTime: time.Unix(1479253692, 0)}
+	info := bindataFileInfo{name: "NAME-service/NAME-cli-client/client_main.gotemplate", size: 2788, mode: os.FileMode(436), modTime: time.Unix(1464111000, 0)}
```

Comment:
yeah, mode is the other one that gets me. If that where added, this'd be perfect!